apps/hip/templates/includes/search_result_card.html

```diff
@@ -14,10 +14,17 @@
         </div>
         <div class="level-right">
           {% if result.specific.get_priority_display %}
+            {# Health alert pages show the icon for their priority and their priority #}
```

Comment:
```suggestion
            {# Health alert pages show the icon for their priority  #}
```

Reply:
You may have intended `and their priority`; I wasn't certain though.

Reply:
I did actually mean to include both (their priority and the icon for their priority). I'll try to reword it.